File: Python Code/Daily_Flash_Code_0/Week 3/31Jan/16-DailyFlash_Solutions/16-DailyFlash_Solutions/31_Jan_Solutions_Two/Java/progam2.java
import java.io.*;

class Template {
	public static void main(String[] args) throws IOException {
		
		float n = 0;
		System.out.printf("Rupees : ");
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		
		try {
			n = Float.parseFloat(br.readLine());
		}catch(NumberFormatException nf){
			System.exit(0);
		}
		System.out.printf("Dollars : %f\n", n * 0.014);
	}
}

File: Python Code/Daily_Flash_Code_0/Week 3/31Jan/16-DailyFlash_Solutions/16-DailyFlash_Solutions/31_Jan_Solutions_Two/C/progam2.c

#include <stdio.h>
#include <stdlib.h>

int main() {

	float n;
	printf("Rupees : ");
	scanf("%f", &n);
	
	if(n <= 0)
		exit(0);

	printf("Dollars : %f\n", n * 0.014);
}
